capa/rules/__init__.py

```diff
@@ -940,6 +940,12 @@ def _extract_subscope_rules_rec(self, statement):
             for child in statement.get_children():
                 yield from self._extract_subscope_rules_rec(child)
 
+    def is_internal_rule(self) -> bool:
```

Comment:
this function seems to be unused right now

Reply:
yeah, it was included in main.py as well. should I get rid of it?